cioppino/twothumbs/browser/templates/login_to_rate.pt

```diff
@@ -1,10 +1,3 @@
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en"
-      xmlns:tal="http://xml.zope.org/namespaces/tal"
-      xmlns:metal="http://xml.zope.org/namespaces/metal"
-      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
-      lang="en"
-      i18n:domain="cioppino.twothumbs">
```

Comment:
stripping out the i18n domain is not what you want.
what about keeping this with namespaces and i18n domain and just setting an omit-tag on html?